src/shared/store/posts-ui/hook.ts

```diff
@@ -0,0 +1,5 @@
+import { usePostsUIStore } from './store'
+
+export function usePostsUI() {
+  return usePostsUIStore()
+}
```

Comment:
어떤 의미가 있는 코드인지 모르겠어요, 네이밍이 문제라면 차라리 이건 어떠신지?

```suggestion
export { usePostsUIStore as usePostsUI } from './store'
```